### ISSUE TITLE
Reorder between multiple UIStackViews 

### ISSUE THREAD

**Issue by ghost** [clayellis/APReorderableStackView#3]
Thank you for your awesome work! 
I need to  reordering multiple stackView, similar to following case 
UIStackView 1 ---> UIView 1 
UIStackView 2 ---> UIView 2 
Where I need to reorder UIView1 and UIView 2

Is this something that I can achieve with this lib ? I'm already setting up a custom stackView but wanted to know if I can extend yours and make PR. any thoughts that can point to right direction...

**Comment by clayellis** [comment]
You mean taking an `arrangedSubview` from one stack view and placing it in another?

**Comment by ghost** [comment]
Yes. I can reorder in the same StackView but not between two StackViews.

**Comment by clayellis** [comment]
I don't believe that's supported yet but if you want to give this a shot and submit a PR that would be awesome. If not I'll try to take a look at in the next few weeks.
